### DISCUSSION TITLE
Priiloader 0.10.0 RC3  - Testing and Info - Testing, Expanding and Fixing of Old Wii Hacks, vWii Support Continues

### DISCUSSION THREAD

**Discussion by Ingunar** [DacoTaco/priiloader#339] [General]
### Priiloader is so very close to the 0.10.0 milestone!

**We use this note to keep track of the current tests, bugs, and progress on Priiloader's development.**
**You can find more info about the hacks and testing in the note:**
[**Priiloader 0.10.0 - RC3 - Testing and Info**](https://hackmd.io/@Ingunar/PriiloaderTesting)

### **We need some help with testing some NEW and especially OLDER hacks for all of the Wii, vWii, and Mini versions, regions, and revisions!**

And of course...
***Everyone is welcome to help!***

**Comment by Ingunar** [comment]
### **IMPORTANT INFORMATION**


**It has been reported that if the Priiloader is installed, using MyMenuify to change vWii System Menu theme can cause brick!**

MyMenuify, same as USB Loader GX, needs to be updated to work with vWii Priiloader.
Be careful if you are using any app that changes/works with the Wii System Menu asset archive.
If you know any similar app that should be checked, tell me.

**Comment by Ingunar** [comment]
**Priiloader 0.10.0 RC3 Testing - Known common issues and Info:**

__Known issues:__

- ~~**Gecko Dumping to SD in some very rare cases doesn't work.**~~ - Aroma is causing this somehow. When Auto booting to SM while booting from Aroma via the Wii Menu icon.

__Wii System Settings on vWii - Additional info:__

- Currently, anything changed inside the System settings will be reset to the previous settings after the console's reboot.
This is normal vWii behavior.
Almost every setting can be changed permanently inside the WiiU's settings.
New hacks to resolve this are planned to be created in the future.
You can find full info in the Testing note's: "Explanation about the Wii System Settings and the permanent SYSCONF solution" section.

__Other Info:__

- ~~**BootMii** iOS doesn't exist for vWii~~ - It probably never will. This option can not be currently used with vWii
- `Ignore standby`  and `Light slot on Error` in Priiloader options have no use for vWii
- ~~On vWii, USB Loader GX currently has graphical glitches with the Channel Banners~~ - The developer fixed it, and the update for it will be released soon!
- **Using MyMenuify to change vWii System Menu theme will cause brick.** 
(The app will need an update to work with Priiloader, but can be used to change the theme before the Priiloader is installed)

__Hacks support is still incomplete. Detailed information can be found in [the note](https://hackmd.io/@Ingunar/PriiloaderTesting)__

**Reply by KendrickLamar2003** [reply]
No new releases of USB Loader GX have been released yet, and its been more than a month since you posted this!.

**Reply by DacoTaco** [reply]
@swaggio7 : that is USB Loader GX's issue tbh. we have made them aware and they have know what and how to fix it

**Reply by Ingunar** [reply]
To be able to load the games before the USB Loader GX update comes out, you must go to the loader's GUI settings and set `Game Window Mode` to `Rotating Disc` (Which allows you to access game settings and start the game)

**Reply by KendrickLamar2003** [reply]
> To be able to load the games before the USB Loader GX update comes out, you must go to the loader's GUI settings and set `Game Window Mode` to `Rotating Disc` (Which allows you to access game settings and start the game)

Fun Fact: There is no need for this. The buttons to start games and change their setting exists and are clickable, they just don't appear visually.

**Reply by DacoTaco** [reply]
doesn't change the fact they need to fix it though

**Reply by AdRoz78** [reply]
1 year later and still haven't fixed it

**Reply by Ingunar** [reply]
> 1 year later and still haven't fixed it

USBLGX dev fixed it but didn't release it
And he has been missing in action since January, so... ¯\_(ツ)_/¯

**Comment by Ingunar** [comment]
### You can generate the vWii Timestamp Fix Hack for your specific vWii and timezone here:

[**vWii Priiloader WC24 UTC Patch Generator**](https://garyodernichts.github.io/priiloader-patch-gen/)

**Comment by Ingunar** [comment]
**New `hacks_hash.ini` for testing added:**

Version [0.10.0 RC3 - Test 02](https://cdn.discordapp.com/attachments/1078641159510958090/1139984956513341440/hacks_hash.ini)

- FIX [No System Menu Background Music] - Wii, Wii Mini

Version 0.10.0 RC3 - Test 01

- Re-enabled [Disable SYSCONF overwrite on boot] for vWii
- Re-enabled [Remove IOS16 Disc Error] for vWii
- Re-enabled [Allow TitleID RAAE,408x,410x] for vWii
- Re-enabled [Block Disc Autoboot] for vWii
- Re-enabled [No System Menu Sounds AT ALL] for all

**Comment by DacoTaco** [comment]
@Ingunar : shouldn't that hacks file be on the repo instead?

**Reply by Ingunar** [reply]
Nah, this file is testing only. I enabled untested vWii ones and the experimental vWii SYSCONF one. 
SYSCONF one can break the other one in some cases, that is why we didn't add it.

[No System Menu Background Music] changes can be added to the repo.

**Reply by Ingunar** [reply]
vWii ones that got Re-enabled are not fully ported and tested, also not really needed or can not be properly used and tested at the moment, 
That is why they got cut from the 0.10.0 RC3 release, but I enabled it for testing if someone finds a creative way to test it...

**Comment by Ingunar** [comment]
### Hey people, the Priiloader 0.10.0 Final will be released very soon!

I want to do a final hack check before we push it, to confirm that all of the remaining hacks are working properly.
If anyone is willing to help, we need someone who owns USA Wii and/or JAP Wii with the System Menu 4.3 installed to test the hacks.

Every little bit of the help is much appreciated! 😄

**Comment by Ingunar** [comment]
# Priiloader 0.10.0 Final has been released!
This discussion will stay open for a bit more, and the special hack testing discussion will be created after that.
@DacoTaco will create new testing discussions for the app from now on. (If those are ever needed again)

### 🥳 Thanks, everyone for the reports and testing! 🎈 
### 🦖 **This was a huge milestone for us and the whole community!** 🫂



**Also, `hacks_hash.ini` for testing was updated:**

Version [0.10.0 - Test 01b](https://cdn.discordapp.com/attachments/1078641159510958090/1150094894837268520/hacks_hash.ini)

- REMOVED incompatible hacks for Wii Mini - [Remove NoCopy Save File Protection], [Re-Enable Bannerbomb v2], [Mark Network Connection as Tested] - SD and WiFi cards mods require 51x version of System Menu
- Fixed some minor stuff
- Reuploaded as `Test 01b` to fix an invalid line caused by a typo 😢 

Just a reminder that you can always check the [**PRIILOADER HACKS TESTING NOTE**](https://hackmd.io/@Ingunar/PriiloaderTesting) to see the progress.
